pkg/services/alerting/notifiers/slack.go

```diff
@@ -73,25 +75,36 @@ func init() {
         </info-popover>
       </div>
       <div class="gf-form max-width-30">
-        <span class="gf-form-label width-6">Mention</span>
+        <span class="gf-form-label width-8">Mention User</span>
         <input type="text"
           class="gf-form-input max-width-30"
-          ng-model="ctrl.model.settings.mention"
+          ng-model="ctrl.model.settings.mentionUser"
           data-placement="right">
         </input>
         <info-popover mode="right-absolute">
-          Mention a user or a group using @ when notifying in a channel
+          Mention a user when notifying in a channel
```

Comment:
Need to extend this I think to include that you must reference user id's and how to include multiple

Reply:
@marefr Oh yeah thanks, slipped my mind.

Reply:
@marefr I didn't think of supporting multiple user IDs, what should we use for separator? Space, comma?